### ISSUE TITLE
AMD A6-3400M))))

### ISSUE THREAD

**Issue by roma323** [exelban/tensorflow-cifar-10#5]
AMD A6-3400M 
Epoch 1 - accuracy: 39.43% (3943/10000) - time: 00:57:37.07 )))

**Comment by exelban** [comment]
@roma323 You need to reduce batch size, maybe increase number of epoch and adjust learning rate.
